include/VecCore/Backend/Libmvec.h

```diff
@@ -0,0 +1,131 @@
+#define FORCE_INLINE inline __attribute__((always_inline))
+
+#ifndef VECCORE_ENABLE_LIBMVEC
+#define VECCORE_ENABLE_LIBMVEC
+
+#include <math.h>
+
+template <typename T>
+FORCE_INLINE
+T *Frexp(const T *x, const T *exp)
+{
+  int N = *(&x + 1) - x; 
+  T *result = new T[N];
+  
+  #pragma omp simd
+  for(int i = 0;i < N;i++)
+  {
+    result[i] = frexp(x[i], &exp[i]);
+  }
+  
+  return result;
+}
+
+template <typename T>
+FORCE_INLINE
+T *Deg(const T *x)
+{
+  int N = *(&x + 1) - x;
```

Comment:
By the way, you need to use `VectorSize<T>()` to get the size, the `N` you will get above will be wrong for all SIMD types.